src/libraries/System.Collections.Immutable/src/Interfaces.cd

```diff
@@ -1,52 +0,0 @@
-﻿<ClassDiagram MajorVersion="1" MinorVersion="1">
-  <Interface Name="System.Collections.Generic.IHashKeyCollection&lt;TKey&gt;">
```

Comment:
What are .cd files, out of curiosity?

Reply:
They're associated with the dgml features in VS. I suspect these got accidentally checked in 10 years ago when the library was initially added.